dags/inference/configs/jetstream_pytorch_gce_config.py

```diff
@@ -83,22 +83,26 @@ def get_jetstream_pytorch_inference_nightly_config(
       f"export CKPT_PATH={model_configs['checkpoint']}",
       f"export TOKENIZER_PATH=$(pwd)/ckpt_dir/{model_configs['tokenizer']}",
       f"export SHARDING_CONFIG={model_configs['sharding_config']}",
-      "mkdir ckpt_dir",
-      "gsutil cp -r ${CKPT_PATH}/* ckpt_dir/",
+      f"export QUANTIZE={str(model_configs['quantize'])}",
+      "mkdir /dev/shm/ckpt_dir",
```

Comment:
Curious, how is this different from `mkdir ckpt_dir`?

Reply:
llama2-70b can't hold on the default disk

Reply:
Understood, thanks!